src/ai/susi/mind/SusiSkill.java

```diff
@@ -145,6 +145,18 @@ else if (bang_type.equals("console")) {
                                             boa.has("latitude") && boa.has("longitude") && boa.has("zoom")) {
                                         actions.put(SusiAction.mapAction(
                                             boa.getDouble("latitude"), boa.getDouble("longitude"), boa.getInt("zoom")));
+                                    } else
+                                    if(type.equals(SusiAction.RenderType.alarm_set.toString()) &&
+                                            boa.has("hour")){
+                                        int hour = boa.getInt("hour");
+                                        if(hour>12)
+                                            actions.put(SusiAction.alarmSetAction(
+                                                    hour-12, boa.has("minute") ? boa.getInt("minute") : 0,
```

Comment:
I don't think that hour-12 is a good idea, do you think that this should actually work? How do you set am or pm?